textract-lambda-cdk-dotnet/Cdk/src/Cdk/CdkStack.cs

```diff
@@ -0,0 +1,81 @@
+using Amazon.CDK;
+using Amazon.CDK.AWS.IAM;
+using Amazon.CDK.AWS.Lambda;
+using Amazon.CDK.AWS.Lambda.EventSources;
+using Amazon.CDK.AWS.Logs;
+using Amazon.CDK.AWS.S3;
+using Constructs;
+
+namespace Cdk
+{
+    public class CdkStack : Stack
+    {
+        readonly string lambdaFunctionName = "textractLambdaFunction";
+        internal CdkStack(Construct scope, string id, IStackProps props = null) : base(scope, id, props)
+        {
+            //Create S3 bucket
```

Comment:
I appreciate the comment but it doesn't add any value in my opinion. I suggest removing them and adding comments wherever something might not be immediately obvious

Reply:
Done. Remove comments